test/sql/include/join_load.sql

```diff
@@ -0,0 +1,146 @@
+-- This file and its contents are licensed under the Apache License 2.0.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-APACHE for a copy of the license.
+
+
```

Comment:
nit: add a comment that this is taken and modified from pg source just to give credit where it is due.